### DISCUSSION TITLE
Cannot pin columns with column virtualization

### DISCUSSION THREAD

**Discussion by Gumichocopengin8** [TanStack/table#5433] [Q&A]
[Example Code](https://stackblitz.com/edit/vitejs-vite-bqxvgg?file=README.md). 

I use `@tanstack/react-virtual` for virtualization.
The first column doesn't sticky to the left side. I understand this happens because there's column virtualization, but are there neat solutions to make it work? 

https://github.com/TanStack/table/assets/34010665/9c6841e2-52ad-4a52-9d24-980f11a80463

**Comment by expiohm** [comment]
@Gumichocopengin8 u got any fix for this

**Reply by Gumichocopengin8** [reply]
Unfortunately nope
